src/crawlee/_cli.py

```diff
@@ -92,26 +114,66 @@ def _prompt_for_template() -> str:
 def create(
     project_name: Optional[str] = typer.Argument(
         default=None,
+        show_default=False,
         help='The name of the project and the directory that will be created to contain it. '
         'If none is given, you will be prompted.',
+    ),
+    crawler_type: Optional[str] = typer.Option(
+        None,
+        '--crawler-type',
+        '--template',
         show_default=False,
+        help='The library that will be used for crawling in your crawler. If none is given, you will be prompted.',
     ),
-    template: Optional[str] = typer.Option(
+    package_manager: Optional[str] = typer.Option(
         default=None,
-        help='The template to be used to create the project. If none is given, you will be prompted.',
         show_default=False,
+        help='Package manager to be used in the new project. If none is given, you will be prompted.',
+    ),
+    start_url: Optional[str] = typer.Option(
```

Comment:
`Optional` and not `|` because of Typer, am I correct? thanks

Reply:
Yes. Typer and Python 3.9, IIRC.